### ISSUE TITLE
Import in non-package issue

### ISSUE THREAD

**Issue by DigiAngel** [decalage2/oletools#139]
Topic:
~~~
Traceback (most recent call last):
  File "/opt/oletools/oletools/olevba.py", line 259, in <module>
    from .thirdparty import olefile
ValueError: Attempted relative import in non-package
~~~
This is from the latest git this morning.  Anything I can do to troubleshoot this?  Thank you.

**Comment by decalage2** [comment]
This is due to recent changes in the code (PR #138 by @Rafiot). A python script, when run directly instead of being imported as a package, cannot use relative imports. I will need to fix this issue.

Two workarounds to this:
- either reinstall oletools using pip (https://github.com/decalage2/oletools/wiki/Install#how-to-install-the-latest-development-version), and then simply use "olevba" to run it from a shell. This is a shortcut which imports olevba.py and runs it. It should work.
- or reinstall the previous development version manually (commit 879c3fb44448a0b7176ff317f6c8737837c39436)

**Comment by Rafiot** [comment]
Woops, sorry, feel free to assign it to me too, I can try to fix it in the next days.

**Comment by DigiAngel** [comment]
Thank you...I got what I needed running an older version on a different box, so I'm good for now...thanks for the quick response on this.

**Comment by decalage2** [comment]
Fixed by commits cd61e4ff9d0089e037ef4f1b1c6f23d8562ea722 and 65449b94e52c10c773a9e6a8f07e654097faae9e.
